src/test/lib/storage/constraints/table_key_constraint_test.cpp

```diff
@@ -145,6 +142,9 @@ TEST_F(TableKeyConstraintTest, Less) {
   EXPECT_FALSE(key_constraint_a < key_constraint_a_reordered);
 
   EXPECT_TRUE(primary_key_constraint_a < key_constraint_a);
+  EXPECT_TRUE(primary_key_constraint_a < key_constraint_a_reordered);
+  EXPECT_TRUE(primary_key_constraint_a < key_constraint_b);
+  EXPECT_TRUE(primary_key_constraint_a < key_constraint_c);
```

Comment:
I think we should also add a check for two constraints with the following properties:
`TableKeyConstraint{{<column c1>}, <contraint type t1>}` and `TableKeyConstraint{{<column c1>, <column c2>}, <contraint type t1>}`

Reply:
This is exactly what L151 does:
```c++
const auto key_constraint_a = TableKeyConstraint{{ColumnID{0}, ColumnID{2}}, KeyConstraintType::UNIQUE};
...
const auto key_constraint_c = TableKeyConstraint{{ColumnID{0}}, KeyConstraintType::UNIQUE};
...
EXPECT_TRUE(key_constraint_c < key_constraint_a);
```

Do you want the same for `KeyConstraintType::PRIMARY_KEY`?

Reply:
The line was hidden in the diff, i.e., not expanded. No, I'm fine with it, thank you.